### DISCUSSION TITLE
Restrict public access to log-viewer page when in Production

### DISCUSSION THREAD

**Discussion by itxshakil** [opcodesio/log-viewer#6] [Ideas]
If we're on production mode, There must be a way to restrict unauthorised access to log-viewer route.

I have three ideas in the mind.

- Custom log viewer URL
- Restrict route by HTTP Authentication
- Functionality to Add middleware for route

**Comment by itxshakil** [comment]
I can see it is two of the proposed ideas has been already implemented. 

My Bad.

**Comment by arukompas** [comment]
hey @itxshakil , great ideas!

There isn't much that can't be solved with middleware 😄 

Also, check out the post made by Snapey here - https://github.com/opcodesio/log-viewer/discussions/16
